### PR TITLE
Set correct installment price when swatcher changed

### PR DESCRIPTION
**Pull request by hsour**

### CODE REVIEW

File: Splitit/Paymentmethod/view/frontend/web/js/swatch-renderer-mixin.js [hunk 1]
```diff
@@ -10,12 +10,13 @@ define([
                 var prices = this._getNewPrices();
                 var newPrice = {detail: prices.finalPrice.amount};
                 var oldPrice = window.splitit_product_price;
-                if (oldPrice != undefined && oldPrice != newPrice.detail) {
+                if (oldPrice != undefined) {
```
Comment (hsour) [inline Splitit/Paymentmethod/view/frontend/web/js/swatch-renderer-mixin.js:13]: Why did you remove validation with old and new prices? Without it, this code will be executed each time, when swatches selected. Even if the price is not changed.

File: Splitit/Paymentmethod/view/frontend/web/js/swatch-renderer-mixin.js [hunk 2]
```diff
@@ -10,12 +10,13 @@ define([
...
                     var installments = (newPrice.detail/parseFloat(window.splitit_installments)).toFixed(2);
-                    var textBlock = jQuery('.cart-installment').html();
+                    /* var textBlock = jQuery('.cart-installment').html(); */
                     var newValue = window.splitit_currency + installments;
-                    textBlock = textBlock.replace(/\$([0-9]+\.?[0-9]+)/, newValue);
-                    jQuery('.cart-installment').html(textBlock);
+                    /* textBlock = textBlock.replace(/\$([0-9]+\.?[0-9]+)/, newValue);
```
Comment (hsour) [inline Splitit/Paymentmethod/view/frontend/web/js/swatch-renderer-mixin.js:17]: I think it's better to remove it. We will not use this code when a new selector was added.

File: Splitit/Paymentmethod/view/frontend/web/js/swatch-renderer-mixin.js (hunk 1 above)
Comment (sixg) [reply Splitit/Paymentmethod/view/frontend/web/js/swatch-renderer-mixin.js:13]: yeah I understand that but due to this we were not showing the price for the first variant which is shown by default for a configurable product.
That's why removed it since we need to update the price again if user deselects the attribute or changes the attribute to the first one.

File: Splitit/Paymentmethod/view/frontend/web/js/swatch-renderer-mixin.js (hunk 2 above)
Comment (sixg) [reply Splitit/Paymentmethod/view/frontend/web/js/swatch-renderer-mixin.js:17]: ok will update the PR

**Review by sixg — APPROVED** [review]
